skiko/src/commonTest/kotlin/org/jetbrains/skiko/paragraph/ParagraphStyleTests.kt

```diff
@@ -55,4 +55,13 @@ class ParagraphStyleTests {
             assertEquals(gloriousRasterSettings, paragraphStyle.fontRastrSettings)
         }
     }
+
+    @Test
+    fun paragraphStyleRoundingHackTests() {
+        ParagraphStyle().use { paragraphStyle ->
```

Comment:
Changing this one to this would make k/js happy :D

```kotlin
    @Test
    fun paragraphStyleRoundingHackTests() {
        ParagraphStyle().use { paragraphStyle ->
            assertFalse(paragraphStyle.isApplyRoundingHackEnabled)
            paragraphStyle.isApplyRoundingHackEnabled = true
            assertTrue(paragraphStyle.isApplyRoundingHackEnabled)
        }
    }
```
`_nGetApplyRoundingHack` returns 0 (there is no false/true in wasm). And JS equality comparision fails:
`0 === false --> false` 
___

Not using "add a suggestion" because new imports are needed anyway.

Reply:
jfyi: to run the k/js tests locally:

```
./gradlew :cleanJsBrowserTest :jsBrowserTest -Pskiko.wasm.enabled=true -Pskiko.js.enabled=true
```

Reply:
Thanks!